From e3b12e0d518228b48e4b642adc52498fbb4b61c2 Mon Sep 17 00:00:00 2001
From: LiGang <b41990@freescale.com>
Date: Sat, 5 Jan 2013 17:01:53 +0800
Subject: [PATCH 33/53] ENGR00237588: Add PTP ctrl request func

PC maybe issue out MTP_REQ_GET_DEVICE_STATUS control request to query the
device status when device in PTP mode. So it's necessary to add ctrl request
function for PTP.

Signed-off-by: LiGang <b41990@freescale.com>
---
 drivers/usb/gadget/android.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/gadget/android.c b/drivers/usb/gadget/android.c
index c9e3dd9..7e3b161 100644
--- a/drivers/usb/gadget/android.c
+++ b/drivers/usb/gadget/android.c
@@ -336,6 +336,7 @@ static struct android_usb_function ptp_function = {
 	.init		= ptp_function_init,
 	.cleanup	= ptp_function_cleanup,
 	.bind_config	= ptp_function_bind_config,
+	.ctrlrequest	= mtp_function_ctrlrequest,
 };
 
 
-- 
1.7.10.4

